assembl/views/api2/auth.py

```diff
@@ -368,31 +368,30 @@ def reset_password(request):
                     localizer.translate(_("This email does not exist")),
                     code=HTTPNotFound.code)
             else:
-                raise JSONError(
-                    localizer.translate(_(generic_error_message)),
-                    code=HTTPNotFound.code)
+                send_change_password_email(request, user, email, discussion=discussion)
```

Comment:
Why do you call `send_change_password_email` if this will do nothing?

Reply:
`return HTTPOk()` instead of calling a function that does nothing

Reply:
the requested change is to show the message 'We have sent you an email with a temporary connection link. Please use that link to log in and change your password.' So yes it does nothing in the case the email does not exist in the database, and it shows the message. so this is coherent with the logic of the task.

Reply:
I agree with @vincentfretin here. Instead of removing the error and raising another Exception downstream in the `send_change_password_email`, which would fail quite early in the same thread, causing an exception error to be shown (exact opposite of what you're trying to do). In these circumstances, you should consume the error, log it as an error in Assembl, so that we can trace people who tried to bang at this API.